source/includes/container/common-minio-kes.rst

```diff
@@ -69,33 +69,27 @@ The following commands create two TLS certificates that expire within 30 days of
     :class: copyable
     :substitutions:
 
+    # These commands output keys to |path|/certs on the host operating system
```

Comment:
`|path|` needs to be corrected. It is not replacing correctly in the rendered file.

Reply:
Done

Also: `\s*\\` <- regex to replace all of those kinds of code-block style things. Fun.